src/components/organization/dto/organization-type.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { ObjectType, registerEnumType } from '@nestjs/graphql';
+import { SecuredEnum } from '../../../common';
+
+/**
+ * The type of organization.
+ */
+export enum OrganizationType {
+  Church = 'Church',
+  Parachurch = 'Parachurch',
+  Mission = 'Mission',
+  TranslationOrganization = 'TranslationOrganization',
+  Alliance = 'Alliance',
+}
+
+registerEnumType(OrganizationType, {
+  name: 'OrganizationType',
+  description: 'The type of organization',
```

Comment:
Customize the label like this:
```suggestion
  description: 'The type of organization',
  valuesMap: {
    Parachurch: {
      description: `@label Para-church`,
    },
  },
```

Reply:
So this would be used only if we wanted the label to look different than the value?

Reply:
Yeah if it doesn't follow the standard convention you can customize like this. This is just put in the GQL schema. The UI repo looks for this and uses it as an override instead of doing the standard transformation.
https://github.com/SeedCompany/cord-field/blob/develop/src/api/schema/enumLists/enumLists.codegen.ts#L44-L44
https://github.com/SeedCompany/cord-field/blob/develop/src/api/schema/enumLists/enumLists.codegen.ts#L57-L59

Reply:
Cool.  In this case we don't need it, because the actual spelling of the word here is "Parachurch" with no hyphen; it's just wrong on the mock-up.

Reply:
Ah cool.